services/wallet/price_test.go

```diff
@@ -0,0 +1,72 @@
+package wallet
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/status-im/status-go/appdatabase"
+	"github.com/status-im/status-go/sqlite"
+)
+
+func setupTestPriceDB(t *testing.T) (*PriceManager, func()) {
+	tmpfile, err := ioutil.TempFile("", "wallet-price-tests-")
+	require.NoError(t, err)
+	db, err := appdatabase.InitializeDB(tmpfile.Name(), "wallet-prices-tests", sqlite.ReducedKDFIterationsNumber)
+	require.NoError(t, err)
+	return &PriceManager{db}, func() {
+		require.NoError(t, db.Close())
+		require.NoError(t, os.Remove(tmpfile.Name()))
+	}
+}
+
+func TestPrice(t *testing.T) {
+	manager, stop := setupTestPriceDB(t)
+	defer stop()
+
+	rst, err := manager.GetCachedPrices()
+	require.NoError(t, err)
+	require.Empty(t, rst)
```

Comment:
Reminder, if the query fails, this will be `nil`.

Reply:
That's why we have line 30, if it fails we won't get to line 31, right?